### PR TITLE
[WIP] Add rake task db:anonymize_admissions

### PR DESCRIPTION
**Pull request by alfiehub**
Usage: `rake db:anonymize_admissions`

Improvements?

### CODE REVIEW

**Comment by aleksanb** [comment]
How about either emptying the relevant fields, or setting them to "[REDACTED]"?

**Comment by aleksanb** [comment]
It should probably take an admission to censor, that way we can access all the relevant data via @admission.applicants, .job_applications etc.

**Comment by aleksanb** [comment]
We talked:
- The task should always wipe all data
- Users should be deactivated, preventing even a correct guess from logging them in
- Use faker phone number

File: lib/tasks/db.rake [hunk 1]
```diff
@@ -0,0 +1,30 @@
+namespace :db do
+  desc "This task removes all data from the database that can be associated with an applicant."
+  task :anonymize_admissions => :environment do
+    Applicant.where("disabled = ?", false).each do |applicant|
```
Comment (Filip0) [inline lib/tasks/db.rake]: You can use Applicant.where(disabled: false)

File: lib/tasks/db.rake [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        email: Faker::Internet.email,
+        password: password,
+        password_confirmation: password,
+        disabled: true
+      )
+
+      applicant.save(validate: false)
+    end
+
+    Interview.where("comment != ?", "").each do |interview|
```
Comment (Filip0) [inline lib/tasks/db.rake]: Same as above

File: lib/tasks/db.rake [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      interview.assign_attributes(comment: "")
+      interview.save(validate: false)
+    end
+
+    JobApplication.where("motivation != ?", "").each do |job_application|
```
Comment (Filip0) [inline lib/tasks/db.rake]: Same as above

File: app/controllers/applicant_sessions_controller.rb [hunk 4]
```diff
@@ -20,6 +20,9 @@ def create
       @applicant_login_email = params[:applicant_login_email]
       render :new
       return
+    elsif applicant.disabled
```
Comment (Filip0) [inline app/controllers/applicant_sessions_controller.rb]: You can filter the disabled applicants on the authenticate method

**Review by Filip0 — CHANGES_REQUESTED** [review]

File: lib/tasks/db.rake [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+      applicant.save(validate: false)
+    end
+
+    Interview.where(comment: "").each do |interview|
+      interview.assign_attributes(comment: "")
+      interview.save(validate: false)
+    end
+
+    JobApplication.where(motivation: "").each do |job_application|
```
Comment (Filip0) [inline lib/tasks/db.rake]: If motivation is blank make it blank?

**Review by Filip0 — CHANGES_REQUESTED** [review]

**Review by Filip0 — APPROVED** [review]

**Pull request merged**